README.md

```diff
@@ -1,9 +1,10 @@
 # Overview
 
-The "Carbon.AccountManagement" module...
+The "Carbon.AccountManagement" module contains functions to help find and resolve Windows NT identities.
 
 # System Requirements
 
+* Windows OS (Linux & MacOS not supported)

```

Comment:
@splatteredbits I advised @shoreparty to target Windows-only support for version 1.x of this module as the functions we're migrating from Carbon are quite Windows specific. It's going to be significantly more work to port [Carbon/Identity.cs](https://github.com/webmd-health-services/Carbon/blob/develop/Source/Identity.cs) to work cross-platform. I'm not finding any cross-platform .NET native way to resolve users and groups on a system. Do you have any recommendations? My thinking is we'd have to parse the contents of the `/etc/passwd/` and `/etc/group` files to get the information we'd need and call out to the `usermod` command to do the add/remove group operations.

Reply:
This module will contain Windows-only functions, but we can still make sure the module installs, loads, and supports Linux/macOS as much as possible.

For Linux, I can't find any .NET APIs for finding/searching users. We should just parse /etc/passwd and /etc/group to get user and group information.

I was surprised to find out that you can't even connect to a remote AD domain controller in .NET Core. Yikes.

Reply:
Fair enough, that works. I was thinking we were going with the all or nothing approach in terms of cross-platform compatibility within a module.

In that case should `ConvertTo-SecurityIdentifier` explicitly test for the Windows platform and write an error otherwise?

Reply:
In my testing, the security identifier class throws an exception that states it isn't available on the current platform, which we handle with a Write-Error call. I think that should work. Let as much of the code as possible run. Who knows. Maybe in some future version of PowerShell, it will start working on Linux/macOS.